### ISSUE TITLE
Migrate to AzureAD Provider

### ISSUE THREAD

**Issue by nikolay** [pulumi/pulumi#2423]
I keep getting these warnings:

```
Diagnostics:
  azure:ad:Application (aks):
    warning: urn:pulumi:<redacted>::<redacted>::azure:ad/application:Application::aks verification warning: The Azure Active Directory resources have been split out into their own Provider.

    Information on migrating to the new AzureAD Provider can be found here: https://terraform.io/docs/providers/azurerm/guides/migrating-to-azuread.html

    As such the Azure Active Directory resources within the AzureRM Provider are now deprecated and will be removed in v2.0 of the AzureRM Provider.
```

**Comment by andreujuanc** [comment]
I just smashed my head against this one. 
Any clue?

**Comment by florianrusch** [comment]
Some warning here :raised_hand_with_fingers_splayed:

**Comment by monarci** [comment]
Yeah, same warning here. Maybe I'm just not getting it, but the Terraform documentation doesn't make it clear as to what I need to do in Pulumi. Some example code would be really nice. :persevere:

**Comment by lukehoban** [comment]
This is also being tracked in https://github.com/pulumi/pulumi-azure/issues/168.

**Comment by lukehoban** [comment]
This was addressed some time ago - see pulumi/pulumi-azure#168.
